wizeline/project-lab
# Fix Sync From WOS for profiles. [DRAFT]

#### What does this PR do?
- Fixes the error when trying to run the commanc `sync-all-from-wos`  where the sync will fail when creating/updating the profiles info.
- Change the info for some initial profiles on **_seed.ts_** to prevent projects from not being able to load, since some of them don't have an owner when updating froiles from wos


#### Where should the reviewer start?
Check the changes on the files: **_schema.prisma_** and **_seed.ts_**


#### How should this be manually tested?
- Pull the branch locally and try to run the set of initial commands from **_read.me_** file , including the `sync-all-from-wos`
- Check the results when executing `sync-all-from-wos` . It should execute without the error regarding a foreign key.
- Test in the browser
 - Make sure that the  project page still display the cards with the owner's initials, avatarImage or two dashes as undefined correctly on the main page cards.


#### Any background context you want to provide?



#### What are the relevant tickets?
#132 


#### Screenshots
![Captura de Pantalla 2021-12-21 a la(s) 15 07 49](https://user-images.githubusercontent.com/90728303/146998181-66f991c0-e58d-46f2-9001-09984b556cfa.png)


<img width="1514" alt="Captura de Pantalla 2021-12-21 a la(s) 15 13 29" src="https://user-images.githubusercontent.com/90728303/146998509-b22e2271-b28f-4953-83d1-20f87db14469.png">



#### Questions
- The profiles for b.neha, joaquin.bravo, enoc.villa are the only profiles from the seeds files that get remove when running the sync from WOS. Could we fix this if they create their profiles on WOS, if they don't have a profile already ?



#### Checklist
